features/step_definitions/networking.rb

```diff
@@ -1292,3 +1292,60 @@
   default_network = network_operator.default_network(user: admin)
   raise "env doesn't have IPSec enabled" unless default_network["ipsecConfig"]
 end
+
+Given /^I store the hostname from external ids in the#{OPT_SYM} clipboard on the "([^"]*)" node$/ do |cb_ovn_hostname, node_name|
+  ensure_admin_tagged
+  cb_ovn_hostname ||= "ovn_hostname"
+
+  ovsvsctl_cmd = %w(ovs-vsctl list open .)
+  ovn_pod = BushSlicer::Pod.get_labeled("app=ovnkube-node", project: project("openshift-ovn-kubernetes", switch: false), user: admin) { |pod, hash|
+  pod.node_name == node.name
+}.first
+  @result = ovn_pod.exec(*ovsvsctl_cmd, as: admin, container: "ovnkube-node")
+  if @result[:success]
+    ovn_state_output = @result[:response].strip
+  end
+  raise "Failed to execute network command!" unless ovn_state_output != nil
+  cb[cb_ovn_hostname] = ovn_state_output.split("hostname=").last.split(", ovn-bridge-mappings").first
+  logger.info "nodes's corresponding hostname from external_ids is stored in the #{cb_ovn_hostname} clipboard."
+end
+
+Given /^I store the#{OPT_QUOTED} hostname in the#{OPT_SYM} clipboard for the "([^"]*)" node$/ do |type_name,cb_type_hostname,node_name|
+  ensure_admin_tagged
+  cb_type_hostname ||= "type_hostname"
+  case type_name
+  when "short"
+    ovn_cmd=%w(hostname -s)
```

Comment:
we usually have a space around ` = ` ? I'm assuming that is Ruby style.

`    ovn_cmd = %w(hostname -s)`

Reply:
fixed it, thanks!